index.php

```diff
@@ -95,7 +95,7 @@ private function build_index() {
 
 			$uri = trim( sprintf( 
 					'%s/%s',
-					trim( $uri_info['dirname'], '/' ),
+					trim( $uri_info['dirname'], '/\\' ),
```

Comment:
@Pilskalns should we do the same here https://github.com/Pilskalns/orbiter/blob/cc1eeefaed59ebf81732d5d9278c3f0dcd23c6c7/index.php#L106